src/openrct2/peep/peep.c

```diff
@@ -174,6 +174,12 @@ static void peep_update_ride_inspected(sint32 rideIndex);
 
 bool loc_690FD0(rct_peep *peep, uint8 *rideToView, uint8 *rideSeatToView, rct_map_element *esi);
 
+#ifdef DEBUG_DESYNC 
+#define peep_rand() scenario_rand_data(peep)
```

Comment:
`peep` may not be as useful as part of logging, as it's id: https://github.com/OpenRCT2/OpenRCT2/blob/develop/src/openrct2/peep/peep.h#L469